app/controllers/staff/exports/organisations_controller.rb

```diff
@@ -58,6 +58,20 @@ def third_party_project_activities
     render_xml
   end
 
+  def external_income
+    fund = Fund.new(params[:fund_id])
+
+    respond_to do |format|
+      format.csv do
+        export = QuarterlyExternalIncomeExport.new(@organisation, fund.id)
+
+        stream_csv_download(filename: "#{fund.short_name}_#{@organisation.beis_organisation_reference}_external_income.csv", headers: export.headers) do |csv|
```

Comment:
Could we have the filename as a method in the `QuarterlyExternalIncomeExport` class?